p4_constraints/backend/interpreter_test.cc

```diff
@@ -1287,6 +1287,17 @@ TEST(ParseP4RTInteger, ParsesZeroCorrectly) {
   ASSERT_EQ(zero_string.at(0), '\0');
   EXPECT_THAT(ParseP4RTInteger(zero_string), Eq(0));
 }
+
+TEST(ParseP4RTInteger, ParsesTrailingZeroCorrectly) {
+  std::string hex_string = {'\xe0', '\x00', '\x00', '\x00'};
+  uint hex_sum = 0;
+  for (char c : hex_string) {
+    hex_sum <<= 8;
+    hex_sum |= c;
+  }
+  EXPECT_THAT(ParseP4RTInteger(hex_string), Eq(hex_sum));
```

Comment:
Instead of computing `hex_sum`, it would be more readable to just hard code its value directly. You can use hex notation to make that readable: `0xE0'00'00'00U`